WindowsServerDocs/get-started-19/servicing-channels-19.md

```diff
@@ -91,6 +99,10 @@ Microsoft depends on receiving feedback throughout the development process so th
 - If you're using Microsoft Azure, this release should automatically be activated.
 - If you've obtained this release from the Volume Licensing Service Center or Visual Studio Subscriptions, you can activate it by using your Windows Server 2019 CSVLK with your Key Management System (KMS) environment. For more info, see [KMS client setup keys](../get-started/kmsclientkeys.md).
 
+> [!Note]
+> For easier maintenance and management of activation you can use ADBA (Active Directory based activation) for Servers 2012 or later, including Windows Server SAC.
```

Comment:
Cosmetic/readability/codestyle/punctuation: add comma, remove plural S in "Servers 2012", add hyphen in "Active Directory-based"
```suggestion
> For easier maintenance and management of activation, you can use ADBA (Active Directory-based activation) for Server 2012 or later, including Windows Server SAC.
```